src/backend/controllers/PlaylistController.js

```diff
@@ -45,8 +45,11 @@ export const getAllPlaylistsHandler = function (schema, request) {
 export const addNewPlaylistHandler = function (schema, request) {
   const user = requiresAuth.call(this, request);
   if (user) {
-    const { playlist } = JSON.parse(request.requestBody);
-    user.playlists.push({ ...playlist, videos: [], id: uuid() });
+    const playlist = JSON.parse(request.requestBody);
+    const name = playlist.name;
+    const list = playlist.list === undefined ? [] : playlist.list;
```

Comment:
I get the logic why we are doing it. this is making this API call flexible which is really interesting. I am just wondering is there any different way we can write. Just curious about it. This looks cool as well.